### ISSUE TITLE
[BUG] - Cannot download videos from by_hashtag with playwright because of EmptyResponseError

### ISSUE THREAD

**Issue by Goldrest** [davidteather/TikTok-Api#579]
**Describe the bug**

Because of the #434 issue i chosed to use playwright (Selenium seems detected when using by_hashtag). My goal was to download videos from by_hashtag method. I have no problem with getting tiktoks by_trending method and download them.
But i cannot download data from api.get_Video_By_TikTok() when using by_hashtag. The weird thing is that i cannot get video when using by_hastag because of a TikTokApi.exceptions.EmptyResponseError: Empty response from Tiktok ... when using custom_did. But i can get them without custom_did but obvisouly the tiktoks retrieved cannot be downloaded with api.get_Video_By_TikTok() because of an Access Denied.

**The buggy code**

Please insert the code that is throwing errors or is giving you weird unexpected results.

```
verifyFp = 'verify_XXX'
did = str(random.randint(10000, 999999999))
api = TikTokApi.get_instance(custom_verifyFp=verifyFp, use_test_endpoints=True)
tiktoks = api.byHashtag("funny", count = 3, custom_verifyFp=verifyFp, custom_did = did)
data = api.get_Video_By_TikTok(tiktoks[0], custom_verifyFp=verifyFp, custom_did = did)# bytes of the video
with open("0.mp4", 'wb') as output:
    output.write(data) # saves data to the mp4 file
```

**Expected behavior**

Retrieve and download videos from by_hashtag method

**Error Trace**

```
ERROR:root:TikTok response:
Traceback (most recent call last):
  File "AVM_TikTok_2.py", line 545, in <module>
    tiktoks = api.byHashtag("funny", count = 3, custom_verifyFp=verifyFp, custom_did = did)
  File "C:\Users\X\Anaconda3\lib\site-packages\TikTokApi\tiktok.py", line 915, in by_hashtag
    res = self.getData(url=api_url, **kwargs)
  File "C:\Users\X\Anaconda3\lib\site-packages\TikTokApi\tiktok.py", line 283, in get_data
    ) from None
TikTokApi.exceptions.EmptyResponseError: Empty response from Tiktok to https://m.tiktok.com/api/challenge/item_list/?aid=1988&app_name=tiktok_web&device_platform=web&referer=&root_referer=&user_agent=Mozilla%252F5.0%2B%28iPhone%253B%2BCPU%2BiPhone%2BOS%2B12_2%2Blike%2BMac%2BOS%2BX%29%2BAppleWebKit%252F605.1.15%2B%28KHTML%2C%2Blike%2BGecko%29%2BVersion%252F13.0%2BMobile%252F15E148%2BSafari%252F604.1&cookie_enabled=true&screen_width=1029&screen_height=1115&browser_language=&browser_platform=&browser_name=&browser_version=&browser_online=true&ac=4g&timezone_name=&appId=1233&appType=m&isAndroid=False&isMobile=False&isIOS=False&OS=windows&count=3&challengeID=5424&type=3&secUid=&cursor=0&priority_region=&verifyFp=verify_kobqsdpl_QSXJKxZh_pg9u_4Iyr_8Blt_xMoTx3w5ry3y&did=107654595&_signature=_02B4Z6wo00f01rODarAAAIBDqO373H0xHFazkm4AAMx-68
```

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - TikTokApi Version [e.g. 3.9.5]

**Additional context**

In summary :
by_trending() => no problem for retrieve and download
by_hashtag() + no custom_did => can retrieve but not download because of Access Denied
by_hashtag() + custom_did => cannot retrieve with by_hashtag at all because of an TikTokApi.exceptions.EmptyResponseError: Empty response from Tiktok to ...
i test all the combinaison of with or without custom_did in each methode (by_hashtag, api.get_Video_By_TikTok or get_instance) and nothing worked.

**Comment by heiver13** [comment]
Hi, I have the same problem :(

**Comment by GuiAnacleto** [comment]
Hey, me too

**Comment by davidteather** [comment]
Think this should be fixed now.
